datalad/support/annexrepo.py

```diff
@@ -130,6 +130,13 @@ def __init__(self, path, url=None, runner=None,
         batch_size: int, optional
           if specified and >0, instructs annex to batch this many commands before
           annex adds acts on git repository (e.g. adds them them to index for addurl).
+
```

Comment:
do we need empty lines here? (just asking in general so we stay consistent, so far we didn't have any generally, and original numpy 'standard' doesn't: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#docstring-standard )